python/ray/autoscaler/docker.py

```diff
@@ -7,6 +6,8 @@
 
 logger = logging.getLogger(__name__)
 
+DOCKER_MOUNT_PREFIX = "/tmp/ray_tmp_mount"
```

Comment:
Is this path explicitly created as ssh_user?

Reply:
It is implicitly created with each `rsync up` in the `mkdir -p {DOCKER_MOUNT_PREFIX}/file`